<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaterBoard</class>
 <widget class="QMainWindow" name="WaterBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaterBoard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelStudentName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Leerling</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSchool">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>School</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStudentName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSchool">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWatermarkPDF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>PDFs watermerken</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddPDF">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF toevoegen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddDir">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory toevoegen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemovePDF">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF verwijderen</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listOfPDFs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemoveAll">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alles verwijderen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
